Yafc.UI/Rendering/Font.cs

```diff
@@ -6,6 +6,7 @@ namespace Yafc.UI {
     public class Font {
         public static Font header;
         public static Font subheader;
+        public static Font subsubheader;
```

Comment:
It would be good to explain what the `subsubheader` is. Otherwise it's not clear that it is the headers of the recipe-sections. Same goes to `header` and `subheader` when we're at it.